exporters/zipkin/README.md

```diff
@@ -0,0 +1,56 @@
+# Zipkin Exporter for OpenTelemetry C++
+
+## Prerequisite
+
+* [Get Zipkin](https://zipkin.io/pages/quickstart.html)
+
+## Installation
+
+### CMake Install Instructions
+
+Refer to install instructions [INSTALL.md](../../INSTALL.md#building-as-standalone-cmake-project).
+Modify step 2 to create `cmake` build configuration for compiling Zipkin as below:
+
+```console
+   $ cmake -DWITH_ZIPKIN=ON ..
+   -- The C compiler identification is GNU 9.3.0
+   -- The CXX compiler identification is GNU 9.3.0
+   ...
+   -- Configuring done
+   -- Generating done
+   -- Build files have been written to: /home/<user>/source/opentelemetry-cpp/build
+   $
```

Comment:
Build and installation steps are missing?

Reply:
Yes, have gave reference to INSTALL.md above as it would be same as mentioned in that docu. Only step 2 from those instructions needs modification which is mentioned here.